extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -130,6 +135,13 @@ class AmpYoutube extends AMP.BaseElement {
   /** @override */
   buildCallback() {
     this.videoid_ = this.getVideoId_();
+    this.liveChannelid_ = this.getLiveChannelId_();
+
+    if ((this.videoid_ && this.liveChannelid_)
+      || !(this.videoid_ || this.liveChannelid_)) {
+      dev().error(TAG, 'Exactly one of data-videoid or data-live-channel-id'
```

Comment:
nit: variable consistency. There are this.liveChannelid, `this.getLiveChannelId` and `data-live-channelid` and `data-live-channel-id` in the error message.